packages/lesswrong/lib/collections/comments/schema.ts

```diff
@@ -875,7 +875,42 @@ const schema: SchemaType<"Comments"> = {
     canRead: ['guests'],
     canUpdate: ['sunshineRegiment', 'admins'],
     canCreate: ['members', 'sunshineRegiment', 'admins'],
-  } 
+  },
+
+  doppelComments: {
+    // Resolved by doppelComments resolver
+    type: Array,
+    optional: true,
+    canRead: ['guests'],
+    canCreate: ['admins'],
+    canUpdate: ['admins'],
+    hidden: true,
+  },
+
+  'doppelComments.$': {
+    type: String,
+  },
+
+  ownDoppelCommentVote: resolverOnlyField({
+    type: 'DoppelCommentVote',
+    graphQLtype: 'DoppelCommentVote',
+    canRead: ['guests'],
+    resolver: async (comment: DbComment, args: void, context: ResolverContext) => {
+      const { DoppelCommentVotes } = context;
+      return await DoppelCommentVotes.findOne({commentId: comment._id, userId: context.currentUser?._id});
+    },
+  }),
+
+  doppelCommentVotes: resolverOnlyField({
+    type: 'DoppelCommentVote[]',
+    graphQLtype: '[DoppelCommentVote]',
+    canRead: ['guests'],
+    resolver: async (comment: DbComment, args: void, context: ResolverContext) => {
+      const { DoppelCommentVotes } = context;
+      const dcvs = await DoppelCommentVotes.find({commentId: comment._id}).fetch();
+      return await accessFilterMultiple(context.currentUser, DoppelCommentVotes, dcvs.filter(dcv => !!dcv), context);
```

Comment:
I don't think you need to filter `dcvs` before passing it into `accessFilterMultiple`, but if you want to filter the output for values that exist you can use `filterNonnull`